G/Geant4_julia/build_tarballs.jl

```diff
@@ -24,7 +24,8 @@ cmake ${WORKSPACE}/srcdir/Geant4_cxxwrap -B build \
     -DCMAKE_INSTALL_PREFIX=${prefix} \
     -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
     -DCMAKE_BUILD_TYPE=Release \
-    -DCMAKE_CXX_STANDARD=17
+    -DCMAKE_CXX_STANDARD=17 \
+    -DZLIB_ROOT="${prefix}"
```

Comment:
It doesn't look like this option is being used or effective.  One option is to delete the tbd file in the sysroot, but I think this is a cmake issue.